client/application/home/home.scss

```diff
@@ -12,37 +12,42 @@
         max-height: 80%;
         overflow: scroll;
         .room-list {
-            height: auto;
-            width: 14%;
-            background-color: rgba(255,248,220,0.5);
+            margin: auto;
+            height: 25px;
```

Comment:
No, somehow there is a weird gap on the screen so I have a temporarily fix for it.